Index: app/release/output-metadata.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"version\": 3,\r\n  \"artifactType\": {\r\n    \"type\": \"APK\",\r\n    \"kind\": \"Directory\"\r\n  },\r\n  \"applicationId\": \"me.neko.nzhelper\",\r\n  \"variantName\": \"release\",\r\n  \"elements\": [\r\n    {\r\n      \"type\": \"SINGLE\",\r\n      \"filters\": [],\r\n      \"attributes\": [],\r\n      \"versionCode\": 14,\r\n      \"versionName\": \"0.0.5-alpha.r14.e2fb6ce\",\r\n      \"outputFile\": \"NzHelper_v0.0.5-alpha.r14.e2fb6ce.apk\"\r\n    }\r\n  ],\r\n  \"elementType\": \"File\",\r\n  \"baselineProfiles\": [\r\n    {\r\n      \"minApi\": 28,\r\n      \"maxApi\": 30,\r\n      \"baselineProfiles\": [\r\n        \"baselineProfiles/1/NzHelper_v0.0.5-alpha.r14.e2fb6ce.dm\"\r\n      ]\r\n    },\r\n    {\r\n      \"minApi\": 31,\r\n      \"maxApi\": 2147483647,\r\n      \"baselineProfiles\": [\r\n        \"baselineProfiles/0/NzHelper_v0.0.5-alpha.r14.e2fb6ce.dm\"\r\n      ]\r\n    }\r\n  ],\r\n  \"minSdkVersionForDexing\": 26\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/release/output-metadata.json b/app/release/output-metadata.json
--- a/app/release/output-metadata.json	(revision ac7cb9496579921dae020d1ec52d81a989328737)
+++ b/app/release/output-metadata.json	(date 1749805328589)
@@ -11,9 +11,9 @@
       "type": "SINGLE",
       "filters": [],
       "attributes": [],
-      "versionCode": 14,
-      "versionName": "0.0.5-alpha.r14.e2fb6ce",
-      "outputFile": "NzHelper_v0.0.5-alpha.r14.e2fb6ce.apk"
+      "versionCode": 4,
+      "versionName": "0.0.4-alpha",
+      "outputFile": "app-release.apk"
     }
   ],
   "elementType": "File",
@@ -22,14 +22,14 @@
       "minApi": 28,
       "maxApi": 30,
       "baselineProfiles": [
-        "baselineProfiles/1/NzHelper_v0.0.5-alpha.r14.e2fb6ce.dm"
+        "baselineProfiles/1/app-release.dm"
       ]
     },
     {
       "minApi": 31,
       "maxApi": 2147483647,
       "baselineProfiles": [
-        "baselineProfiles/0/NzHelper_v0.0.5-alpha.r14.e2fb6ce.dm"
+        "baselineProfiles/0/app-release.dm"
       ]
     }
   ],
